src/examples/elementary/entry_example.c

```diff
@@ -555,7 +555,10 @@ elm_main(int argc EINA_UNUSED, char **argv EINA_UNUSED)
 
    en = elm_entry_add(win);
    elm_entry_autosave_set(en, EINA_FALSE);
-   elm_entry_file_set(en, "/tmp/da_test.txt", ELM_TEXT_FORMAT_MARKUP_UTF8);
+   const char *tmpdir = eina_environment_tmp_get();
+   char file[PATH_MAX];
+   eina_file_path_join(file, sizeof(file), tmpdir, "da_test.txt");
+   elm_entry_file_set(en, file, ELM_TEXT_FORMAT_MARKUP_UTF8);
```

Comment:
Maybe doing that `eina_mkstemp`-thing magic you did in another diff?

Reply:
Actually this one doesn't receive a template terminating in `XXXXXX`, so I think it would apply. Confirm?

Reply:
Confirm. From the documentation:
```
@param[in] templatename This is a string. It must contain the six characters 'XXXXXX' at the end or directly followed by an extension as in 'prefixXXXXXX.ext'.
```

Reply:
RRRRRRRRRRRight then.